src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -4414,8 +4414,10 @@ private StatementAst UsingStatementRule(Token usingToken)
             {
                 case TokenKind.EndOfInput:
                 case TokenKind.NewLine:
-                // Example, using module ,FooBar
+                // Example: 'using module ,FooBar'
```

Comment:
This comment should be improved (I wrote this code, but had to think again why this case was necessary).

`GetCommandArgument` will successfully return an argument for a unary array argument - but we don't want to allow that syntax with a using statement.